bitnami/spark/Chart.yaml

```diff
@@ -22,4 +22,4 @@ name: spark
 sources:
   - https://github.com/bitnami/bitnami-docker-spark
   - https://spark.apache.org/
-version: 5.1.2
+version: 5.1.3
```

Comment:
Please bump the minor version since you are adding a new feature

Reply:
done